contracts/BatchExchange.sol

```diff
@@ -377,6 +377,25 @@ contract BatchExchange is EpochTokenLocker {
         return IdToAddressBiMap.hasAddress(registeredTokens, addr);
     }
 
+    /** @dev View returning all byte-encoded sell orders for specified user
+      * @param user address of user whose orders are being queried
+      * @param offset uint determining the starting orderIndex
+      * @param pageSize uint determining the count of elements to be viewed
+      * @return encoded bytes representing all orders
+      */
+    function getEncodedUserOrdersPaginated(address user, uint16 offset, uint16 pageSize)
```

Comment:
How about calling this method inside `getEncodedUserOrders` with pageSize length and offset 0?

Reply:
Very good!